lesson_13/task_2/homework_13_2.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
+import logging
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+file_handler = logging.FileHandler('json_shershen.log')
+file_handler.setLevel(logging.DEBUG)
+
+formatter = logging.Formatter('%(asctime)s.%(msecs)03d - %(levelname)s - %(message)s', '%Y-%m-%d %H:%M:%S')
+
+file_handler.setFormatter(formatter)
+logger.addHandler(file_handler)
+
+
+dir_json_path = Path('work_with_json/')
+
+
+def is_all_files_json(dir_path):
+    """Checks files the in given directory for '.json'-extension
+    :param dir_path: path for a given directory
+    """
+
+    all_files = [file for file in dir_path.iterdir() if file.is_file()]
+
+    for file in all_files:
+        file_name = file.name
+        log_message_info = f'Suffix of the {file_name} is ".json"'
+        log_message_error = f'Suffix of the current file is NOT ".json"'
+
+        if file.suffix == '.json':
+            logger.info(log_message_info)
+        else:
+            logger.error(log_message_error)
```

Comment:
the task is about to check if json file is valid
the file could have .json extension but still be invalid

Reply:
Resolved the task 2 in [a033089](https://github.com/EugeneShershen/aqa_hillel_1308/pull/10/commits/a033089598a3e2147189e80bc162381946461953)